pyClient/testZethUtils.py

```diff
@@ -0,0 +1,32 @@
+import zethUtils
```

Comment:
All these python files are not related to this PR. I guess you rebased this one on top of develop and haven't rebased mpc. Would you mind updating mpc so we do not introduce these changes a second time? (it would remove non-related files from the review also)

Reply:
Otherwise we can forget about the branch `mpc` which does not seem to diverge from develop, and this branch will become the target branch of the other mpc related PRs, and will then be the last one to be merged in develop once everything related to the MPC has been incorporated here.

Reply:
Yeah, I hit the same thing.  The mpc branch should already be level with develop, but for some reason these diffs did not update.  I hoped it would update after some time delay, but I guess not.  Perhaps when I next push to this branch it will cull those extra commits..

Reply:
Those extra commits should no longer appear in the diff.